VirtualKeyboard.js

```diff
@@ -288,24 +296,28 @@ class VirtualKeyboard extends Component {
   }
 }
 
+
+const stylePropType = PropTypes.oneOfType([PropTypes.number, PropTypes.object, PropTypes.array, PropTypes.bool]);
```

Comment:
What's the advantage of using this? I tried passing a number and a boolean and it passed the type verification. Shouldn't we only allow objects?

Reply:
It is so allow, for example, the following code:
```
const styles = StyleSheet.create({
  keyboard: {
    backgroundColor: 'black',
    width: '100%',
  },
});

<VirtualKeyboard
  {...props}
  style={[
    styles.keyboard,
    props.style,
    props.showGreen && { backgroundColor: 'green' },
    props.useFullHeight && { height: '100%' },
]}
/>
```